magento-research/magento2-upward-connector
# Frontname Whitelist

## Description
Basing our work off of Experius's PR #12, cherry-pick the frontname whitelist functionality to allow certain routes to pass directly to Magento frontend.

Closes [PWA-101](https://jira.corp.magento.com/browse/PWA-101)

### Steps to Validate
1) Install module on Magento 2.3.3 (preferably with some sample data)
2) Build Venia Concept with this Magento backend
3) Login to admin and go to Stores > Configuration > General > Web
4) Set UPWARD Config File to `upward.yml` in `/dist`, and add `contact` to Frontname Whitelist
5) Navigate to storefront and verify Venia loads
6) Navigate to a Category and Product, verify those routes work as expected
7) Navigate to `/contact` and verify the contact page is loaded and themed to Luma
